packages/opentelemetry-exporter-collector/src/util.ts

```diff
@@ -0,0 +1,205 @@
+/*!
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  spanIdToBase64,
+  hrTimeEndTime,
+  hrTimeToTimeStamp,
+} from '@opentelemetry/core';
+import { ReadableSpan } from '@opentelemetry/tracing';
+import { Attributes, TimedEvent, TraceState } from '@opentelemetry/types';
+import {
+  OTCAttributeMap,
+  OTCAnnotation,
+  OTCAttributes,
+  OTCAttributeValue,
+  OTCSpan,
+  OTCTimeEvent,
+  OTCTimeEvents,
+  OTCTruncatableString,
+  OTCTraceState,
+} from './types';
+
+const OC_MAX_STRING_LENGTH = 128;
+const OC_MAX_ATTRIBUTES = 30;
+
+/**
+ * convert string to maximum length of 128, providing information of truncated bytes
+ * @param name - string to be converted
+ */
+export function stringToTruncatableString(name: string): OTCTruncatableString {
+  const value = name.substr(0, OC_MAX_STRING_LENGTH);
+  const truncatedByteCount =
+    name.length > OC_MAX_STRING_LENGTH ? name.length - OC_MAX_STRING_LENGTH : 0;
+
+  return { value, truncatedByteCount };
+}
+
+/**
+ * convert attributes
+ * @param attributes
+ * @param maxAttributes - default value is 30
+ */
+export function convertAttributesToOTCAttributes(
+  attributes: Attributes,
+  maxAttributes: number = OC_MAX_ATTRIBUTES
```

Comment:
Maybe instead of dropping for now I can add information to `ReadableSpan` to remove this logic once it will be added there. Otherwise if any1 would like to use `CollectorExporter` might find this to be buggy until the `ReadableSpan` supports that.

Reply:
`const OT_MAX_ATTRIBUTES = 30;` is not correct, we have provided options to end users to configure these values as per their application need. See : https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-tracing/src/utility.ts#L33

I agree that it would be buggy in the sense that it would not show the correct count for `droppedAttributeCount`, but AFAIK this is only applicable in case of Stackdriver Exporter, these counts is not being used by other exporters. WDYT?

Reply:
ok so let me remove it them for now